hnn_core/cell_response.py

```diff
@@ -112,9 +112,9 @@ def __init__(self, spike_times=None, spike_gids=None, spike_types=None,
         self._vsoma = list()
         self._isoma = list()
         if times is not None:
-            if not isinstance(times, np.ndarray):
+            if not isinstance(times, (list, np.ndarray)):
                 raise TypeError("'times' is an np.ndarray of simulation times")
-        self._times = times
+        self._times = np.array(times)
```

Comment:
I found it a bit weird that `list` was not accepted here. Made the same change in `Dipole` class

Reply:
We should update the `CellResponse` `times` arg docstring to be array-like